R/map.R

```diff
@@ -156,6 +156,14 @@
 mirai_map <- function(.x, .f, ..., .args = list(), .promise = NULL, .compute = NULL) {
   require_daemons(.compute = .compute, call = environment())
   is.function(.f) || stop(sprintf(._[["function_required"]], typeof(.f)))
+  if (is.null(.compute)) .compute <- .[["cp"]]
+
+  if (otel_tracing) {
+    spn <- otel::start_local_active_span(
+      "mirai::mirai_map",
+      links = list(compute_profile = ..[[.compute]][["otel_span"]])
```

Comment:
Will this work if `.compute` is null?

Subsetting a list with a NULL key makes an error. (I don't know what `..` is in practice)

```
❯❯ list(a = 4)[[NULL]]
Error in list(a = 4)[[NULL]] : 
  attempt to select less than one element in get1index

❯❯ list2env(list(a = 4))[[NULL]]
Error in list2env(list(a = 4))[[NULL]] : 
  wrong arguments for subsetting an environment
```

Reply:
This is fine as it's guarded by ` if (is.null(.compute)) .compute <- .[["cp"]]` directly above it.